front/pages/api/v1/apps/[user]/[sId]/runs/index.ts

```diff
@@ -195,7 +160,7 @@ export default async function handler(req, res) {
             method: "POST",
             headers: {
               "Content-Type": "application/json",
-              "X-Dust-User-Id": appUser.id,
+              "X-Dust-User-Id": appOwner.id.toString(),
```

Comment:
Should be `authUser` here (even if not yet in use). Was incorrect in the original file :+1:

Reply:
👍